ansible_base/resource_registry/migrations/0004_remove_resourcetype_migrated.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 4.2.8 on 2024-03-29 19:51
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('dab_resource_registry', '0003_alter_resource_object_id'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='resourcetype',
+            name='migrated',
+        ),
+    ]
```

Comment:
@john-westcott-iv and I went over this in prior PRs, but we are on a policy to squash test_app migrations.

Reply:
This isn't a test app migration, it's in the resource registry. I'm reluctant to squash dab migrations because we've already deployed it to galaxy.ansible.com

Reply:
Thanks for the info, this is good to know.